samsung-galaxybook.c

```diff
@@ -316,7 +322,7 @@ static int galaxybook_acpi_method(struct samsung_galaxybook *galaxybook, acpi_st
 					purpose_str,
 					method);
 			status = -EIO;
-		} else if (out_obj->buffer.pointer[5] == 0xff) {
+		} else if (out_obj->buffer.pointer[4] == 0xff) {
```

Comment:
Should we maybe check if RFLG != 0xAA rather than RFLG == 0xFF??

Reply:
Hi @rapgenic  from what I remember on this I think it is either pos [5] == 0xff (a "failure") vs pos [4] == 0xaa (a "success"). For me it is find either way but just that I don't think pos [4] == 0xff like you wrote here will ever hit?

Reply:
Actually I only noticed this because I created a test method that should have thrown an error buy went on as if nothing happened... Anyway RFLG (i.e. byte 4) being 0xaa on success seems a point of agreement, so maybe it's the best option. In the next days I'll check the behavior of byte 5 anyway to see if i made a mistake